itests/eth_account_abstraction_test.go

```diff
@@ -24,7 +24,7 @@ import (
 	"github.com/filecoin-project/lotus/itests/kit"
 )
 
-// TestEthAccountAbstraction goes over the account abstraction workflow:
+// TestEthAccountAbstraction goes over the placeholder creation and promotion workflow:
 // - an placeholder is created when it receives a message
 // - the placeholder turns into an EOA when it sends a message
 func TestEthAccountAbstraction(t *testing.T) {
```

Comment:
These tests shouldn't really be called "account abstraction" because we do not have account abstraction yet. We should rename them in a separate cleanup patch.